frontend/lib/src/components/widgets/Checkbox/Checkbox.tsx

```diff
@@ -147,6 +156,16 @@ class Checkbox extends React.PureComponent<Props, State> {
           disabled={disabled}
           onChange={this.onChange}
           aria-label={element.label}
+          checkmarkType={
+            element.type === CheckboxProto.StyleType.TOGGLE
+              ? STYLE_TYPE.toggle
+              : STYLE_TYPE.default
+          }
+          labelPlacement={
+            element.type === CheckboxProto.StyleType.TOGGLE
+              ? LABEL_PLACEMENT.right
+              : undefined
+          }
```

Comment:
Nit: think you can replace this with below since toggle default is `left`, but otherwise default is `right` ([here](https://github.com/uber/baseweb/blob/master/src/checkbox/checkbox.tsx#L120C11-L120C11))
```javascript
labelPlacement={LABEL_PLACEMENT.right}
```

Reply:
Good point. I haven't really looked into how the checkbox needs this, but using `LABEL_PLACEMENT.right` seems to work fine  👍